App/KeychainWrapper/KeychainWrapperDemo/KeychainWrapperDemo/ViewModel/ViewModel.swift

```diff
@@ -0,0 +1,62 @@
+import KeychainWrapper
+import SwiftUI
+
+@Observable
+class ViewModel: Keychainable {
+    
+    private let keychainWrapper: KeychainWrapper
+
+    private(set) var keychainItems:[String:String] = [String:String]()
+
+    init(keychainWrapper: KeychainWrapper) {
+        self.keychainWrapper = keychainWrapper
+        keychainItems = keychainItems.merging(keychainWrapper.allEntries, uniquingKeysWith: {current, _ in return current})
+    }
+    
+    @discardableResult
+    func saveValueToKeychain(_ stringToSave: String, keyValue: String) throws -> Bool {
+        //TODO: Probably there is a way to use OptionalAPI here, but failed trying...
```

Comment:
Please create a ticket for that and assign it to me :)

Reply:
I will do it, but have concerns similar as above.